packages/components/.storybook/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require( 'path' );
+
+module.exports = ( { config } ) => {
+	config.module.rules.push( {
+		test: /\.tsx?$/,
+		include: path.resolve( __dirname, '../src' ),
+		use: [
+			require.resolve( 'ts-loader' ),
```

Comment:
Isn't this the same as just passing `ts-loader`? Why do the resolution here?

Reply:
🤷  wrote it in "hack" mode. Good call, fixed in bef964f1d8b76dc3c655ab5aa0eb977af62e0da0